File: 양성주/2기/1주차/같은 숫자는 싫어.java
import java.util.*;

public class Solution {
    public Stack solution(int []arr) {
        Stack<Integer> answer = new Stack<Integer>();
        answer.add(arr[0]);
        for(int i=1;i<arr.length;i++){
        if(arr[i] == answer.peek()){
            continue;
        }else answer.add(arr[i]);
        }
        return answer;
    }
}

File: 이준혁/2기/3주차/같은숫자는싫어.js
function solution(arr) {
  let answer = [];

  answer[0] = arr[0]; // 초기값은 하나 넣어줘서 인덱스 관리
  for (let i = 1; i < arr.length; i++) { // arr 길이만큼
    if (answer[answer.length - 1] !== arr[i]) { // answer의 마지막 값은 배열의 길이-1로 계산
      answer.push(arr[i]); // 다른 값이 나온 경우에만 push
    }
  }

  return answer;
}
